.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,21 @@
+- [ ] Lint clean (`cmake --build build --target lint 2>/dev/null || cmake --build build 2>&1 | grep -v "^--"`)
+- [ ] If changing the Python API: `python -c "import opticore; print(opticore.__version__)"` works
+- [ ] If adding C++ code: new symbols are exported in `src/bindings.cpp` with nanobind ndarray tag
+- [ ] This PR targets the `main` branch
+
+## Notes for Phase 1 contributors
+
+Phase 1 scope is locked to 6 functions (see ROADMAP.md). Please do not expand scope without an ADR. If your change touches `jaeckel.cpp`, the C++ test reference values, or the tolerance ladder — please ask first.
+
+## Questions?
+
+Open a discussion at https://github.com/vivek-varma/opticore/discussions
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)